public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/url_builder.ts

```diff
@@ -6,33 +6,42 @@ export default class UrlBuilder {
   static buildResourceUri(
     subscriptionId: string,
     resourceGroup: string,
-    metricDefinition: string,
-    resourceName: string,
-    templateSrv: TemplateSrv
+    templateSrv: TemplateSrv,
+    metricNamespace?: string,
+    resourceName?: string
```

Comment:
a resource URI does not require a namespace & resource name (e.g. the URI of a resource group)